libs/coin-evm/src/logic.ts

```diff
@@ -22,6 +22,8 @@ import {
   EvmTransactionEIP1559,
   EvmTransactionLegacy,
 } from "./types";
+import eip55 from "eip55";
+import { log } from "@ledgerhq/logs";
```

Comment:
If possible, external imports should always be at the top compared to local imports

Reply:
Protip, in VS code you can organize imports with the **⇧⌥O** keyboard shortcut 👌 

cf. https://code.visualstudio.com/docs/languages/javascript#_organize-imports

Reply:
This should be a eslint rule then...

https://github.com/import-js/eslint-plugin-import

Reply:
> cf. https://code.visualstudio.com/docs/languages/javascript#_organize-imports

You're going to ruin my pyramids again 😭

Reply:
https://github.com/LedgerHQ/ledger-live/pull/5546/commits/c7b9f6a3a8a5ed2a446eb7f18f8fdeb35c6006fc#diff-58968d2e02e9914b57f440daed3d8112e1b5becc68088e9458acee6eddc50674

Reply:
@lvndry Please don't mixup the 2 subjects. This is a completely unrelated change to the problem you're actually fixing, and should be in a different PR where we can all discuss if the whole team wants that change or not 🙏